### DISCUSSION TITLE
因为更新按钮，我卸载了 Arc 浏览器

### DISCUSSION THREAD

**Discussion by giscus** [darkyzhou/blog-comments#7] [General]
# 因为更新按钮，我卸载了 Arc 浏览器

今天，我卸载了 Arc 浏览器。这是我第一次卸载一个我曾经如此喜欢的产品，而且是因为一个更新按钮。我觉得包括我自己在内的开发者都应该在设计软件更新策略时，切实地站在用户的立场上，权衡他们为了更新要付出的代价以及能够获得的价值。

https://darkyzhou.net/articles/i-uninstalled-arc/

**Comment by B41dr** [comment]
chrome侧边栏插件也挺好用的
